src/services/transaction-service/transaction-service.service.spec.ts

```diff
@@ -2,14 +2,25 @@ import { HttpService } from '@nestjs/axios';
 import { Balance } from './entities/balance.entity';
 import { HttpErrorHandler } from '../errors/http-error-handler';
 import { TransactionService } from './transaction-service.service';
+import { Backbone } from '../../chains/entities/backbone.entity';
```

Comment:
nitpick: This import can be shortened.

Reply:
Fixed in [857b5ab](https://github.com/5afe/safe-client-gateway-nest/pull/30/commits/857b5ab863510c87140f573ba4405527c1914975)